src/main/scala/com/microsoft/ml/spark/explainers/BreezeUtils.scala

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark.explainers
+
+import breeze.linalg.{DenseVector => BDV, SparseVector => BSV, Vector => BV, DenseMatrix => BDM}
+import org.apache.spark.ml.linalg.{Vector => SV, Vectors => SVS, Matrix => SM, Matrices => SMS}
```

Comment:
these abbreviations are confusing, how about just adding Breeze to the breeze.linalg names, and keeping the ml linalg names as is?

Reply:
I dont mind the brevity though ;)

Reply:
ok so I removed all the aliasing for spark.ml.linalg vector classes, but kept the BDV/BSV aliasing for breeze vectors. I also like the brevity especially when the DenseVector class is used a lot.